recipes/pyfizi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyfizi" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a4b47cf057fce63e6d912f87d0e497065436f4b501698543863b5c82970c56f6
+
+build:
+  skip: True  # [win or py!=36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14.5
+    - pandas >=0.23.3
+    - pandas-plink
+    - pip
+    - python
+    - scipy >=1.2.0
+  run:
+    - numpy >=1.14.5
+    - pandas >=0.23.3
+    - pandas-plink
+    - python
+    - scipy >=1.2.0
+
+test:
+  imports:
+    - pyfizi
+  commands:
+    - fizi --help
+
+about:
+  home: https://github.com/bogdanlab/fizi
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  license_file:
```

Comment:
This is required, either the package should come with one or you need to add the license to the PR.

Reply:
I added 'LICENSE' to the conda recipe; I had this earlier but the build failed. There is a LICENSE in the github repo, so I'm not sure if that is how to best link to it.

Reply:
@quattro (It doesn't prevent this from being merged but just so you know)

Technically the files on PyPI are in violation of the licence as they don't include a copy. You can include it automatically in future release tarballs on PyPI if you edit your `setup.cfg` to include:

```
[metadata]
license_files = LICENSE
```

See [here (and the sample project)](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license-txt) for more details.

Reply:
Great thanks. I'll add it to the repo and make sure its included in the next official release. Cheers.